kqueen_ui/blueprints/ui/views.py

```diff
@@ -594,7 +594,8 @@ def handle(self):
             # Append provisioner ID to parameter name to make it unique
             parameters = {
                 k + '__' + provisioner['id']: v
-                for [k, v] in _parameters.items()
+                for [k, v]
```

Comment:
Why do you need to change it? 
It
is
less
readable
don't
you
think
so
?
)